docs/reference/search/field-caps.asciidoc

```diff
@@ -3,15 +3,15 @@
 
 The field capabilities API allows to retrieve the capabilities of fields among multiple indices.
 
-The field capabilities api by default executes on all indices:
+The field capabilities API by default executes on all indices:
 
 [source,js]
 --------------------------------------------------
 GET _field_caps?fields=rating
 --------------------------------------------------
 // CONSOLE
 
-... but the request can also be restricted to specific indices:
+...but the request can also be restricted to specific indices:
```

Comment:
How about omitting the dots altogether, starting a new sentence with "The requestst can...". I think that would be much better.

Reply:
I agree -- good suggestion. I'll make the change.